data/pages.yml

```diff
@@ -180,10 +180,10 @@
   pages:
     - title: "Introduction"
       url: "index"
-    - title: "Acceptance Tests"
-      url: "acceptance"
-    - title: "Unit Testing Basics"
-      url: "unit-testing-basics"
+    - title: "Application Tests"
+      url: "testing-application"
+    - title: "Testing Basics"
+      url: "testing-basics"
```

Comment:
This is important - we need to either a) figure out redirects rather than killing the old links, or b) keep the same urls _for now_ and do the redirect/corrections when guides-app launches. I recommend the 2nd option. URLs are like public API and we should avoid changing them.